Orange/widgets/data/tests/test_owoutliers.py

```diff
@@ -189,6 +190,22 @@ def test_migrate_settings(self):
         self.assertEqual(widget.cov_editor.empirical_covariance, True)
         self.assertEqual(widget.cov_editor.support_fraction, 0.5)
 
+    def test_summary(self):
```

Comment:
A test for this already exist (`test_in_out_summary`) and should be updated if insufficient (instead of writing a new one).